media-sound/teamspeak-client-bin/teamspeak-client-bin-3.0.16.ebuild

```diff
@@ -20,6 +20,7 @@ IUSE="alsa pulseaudio"
 REQUIRED_USE="|| ( alsa pulseaudio )"
 
 RDEPEND="
+	dev-libs/quazip[qt5]
```

Comment:
If someone needs that, then we will have to fix quazip instead.

I've talked to jlec, and please add `:0/0` there.